### PR TITLE
Adds in disappearing messages test turning off

### PR DESCRIPTION
**Pull request by burtonemily**
And delete contact test

### CODE REVIEW

File: tests/automation/disappearing_messages.spec.ts [hunk 1]
```diff
@@ -222,3 +225,153 @@ test_Alice_2W(
...
+        englishStrippedStr('disappearingMessagesTurnedOff')
+          .withArgs({ name: alice.userName })
+          .toString(),
+      ),
+      waitForTestIdWithText(
+        bobWindow1,
+        'disappear-control-message',
+        englishStrippedStr('disappearingMessagesTurnedOffYou').toString(),
+      ),
+    ]);
```
Comment (burtonemily) [inline tests/automation/disappearing_messages.spec.ts:358]: I'm wanting to create a function that separates out the control message checks but thought I would do it in a different PR. There are alot of control messages and its kinda clogging up the test.

File: tests/automation/linked_device_user.spec.ts [hunk 2]
```diff
@@ -336,3 +337,90 @@ test_Alice_2W_Bob_1W(
...
+        window: bobWindow1,
+        strategy: 'data-testid',
+        selector: 'new-conversation-button',
+      }),
+      clickOnElement({
+        window: aliceWindow2,
+        strategy: 'data-testid',
+        selector: 'new-conversation-button',
+      }),
+    ]);
```
Comment (Bilb) [inline tests/automation/linked_device_user.spec.ts:368]: could be a doForAll() call

File: tests/automation/disappearing_messages.spec.ts [hunk 3]
```diff
@@ -222,3 +225,153 @@ test_Alice_2W(
...
+        aliceWindow2,
+        'data-testid',
+        'disappear-messages-type-and-time',
+      ),
+      hasElementBeenDeleted(
+        bobWindow1,
+        'data-testid',
+        'disappear-messages-type-and-time',
+      ),
+    ]);
```
Comment (Bilb) [inline tests/automation/disappearing_messages.spec.ts:375]: could be a doForAll() call

**Review by Bilb — APPROVED** [review]
